doc/tutorials/httpd.md

```diff
@@ -125,6 +127,57 @@ a file as an http response using fast raw device i/o.
   (http-response-file res '(("Content-Type" . "text/plain")) "simpled.ss"))
 ```
 
+#### The `/template` handler
+
+The macros from `std/misc/text` can be used to generate HTML ressponses.
+This handler includes the *template.html* template, which takes a `plist`
+as argument. See `std/misc/text` for details on the macros.
+
+```scheme
+(def (template-handler req res)
+  (def (json-ref key)
+    (let* ((json (read-json (open-input-u8vector (http-request-body req))))
+           (ref (hash-ref json key)))
```

Comment:
this should be `hash-get` otherwise it will throw an error if the key is not present.